src/main/java/core/mvc/JsonView.java

```diff
@@ -3,10 +3,27 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.MediaType;
 
 public class JsonView implements View {
     @Override
     public void render(Map<String, ?> model, HttpServletRequest request, HttpServletResponse response) throws Exception {
+        response.setCharacterEncoding("utf-8");
+        response.setContentType(MediaType.APPLICATION_JSON_VALUE);
+        response.getWriter().write(createBody(model));
+    }
+
+    private String createBody(Map<String, ?> model){
+        if(model.isEmpty()){
```

Comment:
공백, 공백라인 컨벤션 등의 부분들은 습관적으로 반영하시는 것이 좋다고 생각해요 😊
```suggestion
        if (model.isEmpty()) {
```